contracts/TokenIOStorage.sol

```diff
@@ -647,9 +656,87 @@ contract TokenIOStorage is Ownable {
      * @param _address Pointer identifier for value in mapping
      * @return { "uint" : "Returns the uint value associated with the key" }
      */
-    function deleteTokenfxUSDBPSRate(address _address) external view returns(bool success) {
```

Comment:
was missing onlyOwner a bug? Need to doublecheck all similar methods for permissions

Reply:
yes, it was a bug 
I already found I think all bugs. It related only to this fxSDBPSRat set/delete.